Makefile

```diff
@@ -661,6 +661,8 @@ ifeq ($(LINK_ABC),1)
 OBJS += $(PROGRAM_PREFIX)yosys-libabc.a
 endif
 
+unexport CXXFLAGS
```

Comment:
Now that I give this PR another look, it would be good to have a comment here explaining the reason for `unexport`.